implementation.md

```diff
@@ -0,0 +1,22 @@
+# Implementation Recommendations
+
+This document describes implementation recommendations for a STAC API.
+
+## CORS
+
+It is recommended that public APIs advertise a permissive CORS configuration so UIs running on a different domain
+may more easily access them.
+
+APIs should acknowledge pre-flight request headers. In general, these header values should be set on responses:
+
+```
+access-control-allow-origin: *
+access-control-allow-methods: OPTIONS, POST, GET
+access-control-allow-headers: Content-Type
```

Comment:
The Transaction extension uses the `If-Match` header, which we should mention...

Reply:
Added a reference to that.